src/assets/TestFramework/ArmDeploymentInput.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Resources.Models
+{
+    /// <summary> ArmDeployment operation parameters. </summary>
+    public partial class ArmDeploymentInput
```

Comment:
This might have changed as well in an open PR: https://github.com/Azure/azure-sdk-for-net/pull/27934